### ISSUE TITLE
In latest version getting error while translating document 

### ISSUE THREAD

**Issue by HarshadPatel1988** [MicrosoftTranslator/DocumentTranslator#61]
Hi Team,

I was testing the new version for API V3 and Custom Translator Feature. In both cases, with only API key and with Category ID, I am getting following error. Is there something else that I need to setup before using?

Can you please look into this?

![image](https://user-images.githubusercontent.com/24873488/41106723-403a618c-6a8e-11e8-8eef-7ee82163dd71.png)

**Comment by chriswendt1** [comment]
Did you try version 2.0.1? I had added some retry logic.

I did notice some unrecoverable errors (error 503) using Custom Translator customized systems. I Believe it is content-related - the systems fails with certain content. If you can share the failing document with me, that'd be great. If not, that's OK, I'll be debugging with the one document I have.

**Comment by SaBaLoc** [comment]
Thanks a lot @chriswendt1 !
I am afraid I cannot share my document because of my company's privacy policies.
In my case, it seemed to randomly work with some files while not with others, don't think it is content-related, though. I thought first it is due to file naming or size, but I excluded this too. Also, the error occurs also when not using any category ID, so that must be a problem with the tool itself. I did download newest version 2.0.1 yesterday and have had problems since then. Everything went well with previous version (for Translator Text v3 and supporting Hub IDs).

**Comment by chriswendt1** [comment]
OK, thanks. I must have introduced the problem you are seeing then with version 2.01. I'll have a look later today. Can you see whether the failure happens always with the same document or set of documents in each try, and not in others?

**Comment by SaBaLoc** [comment]
Thanks a lot @chriswendt1 .
For now it works or doesn't work always with the same files.

**Comment by chriswendt1** [comment]
OK, thanks. Can you try the following for me: You will have a file that is partially translated to the target language. Can you take the untranslated section, save it in a new file of the same format, and try again, see if that fails or succeeds?

**Comment by SaBaLoc** [comment]
Hi @chriswendt1 ,
sure. I took the partially translated file, saved the untranslated parts to a new document (I am using Word), then re-ran it through Custom Translator and had again a partially translated document. I saved again only the untranslated parts to another new document, re-ran it through Custom Translator and this time it couldn't translate any new part. I am using EN-IT language pair, financial/legal content (not sure if this info helps).
Thanks,
Sara
